papyri/gen.py

```diff
@@ -1800,9 +1804,9 @@ def helper_1(
         elif isinstance(target_item, (FunctionType, builtin_function_or_method)):
             sig: Optional[str]
             try:
-                sig = str(inspect.signature(target_item))
-                sig = qa.split(":")[-1] + sig
-                sig = re.sub("at 0x[0-9a-f]+", "at 0x0000000", sig)
+                sig = str(ObjectSignature(target_item))
+                # sig = qa.split(":")[-1] + sig
+                # sig = re.sub("at 0x[0-9a-f]+", "at 0x0000000", sig)
```

Comment:
Another thing we need to remember is that here we drop the name of the object so we need to fix the rendering of the doc.